driver_unix.go

```diff
@@ -4,11 +4,27 @@ package continuity
 
 import (
 	"fmt"
+	"os"
 	"sort"
+	"syscall"
 
 	"github.com/stevvooe/continuity/sysx"
 )
 
+const (
+	atNoFollow = 0x100
+)
+
+func (d *driver) Unlink(path string) error {
+	return syscall.Unlink(path)
```

Comment:
The intent was to only unlink and remove a file rather than remove a directory. They two remove operations could be joined like they are in the `os` package.